main.py

```diff
@@ -130,14 +132,95 @@ def __main__():
         help='Query String'
     )
 
+    host_parser = parent_subparsers.add_parser(HOST, help='Host a local query service, for testing and developmenr')
```

Comment:
Typo in the help message. I think you meant `development`.

Reply:
Thanks, fixed this one